04_HowTos/QueryClient/How_to_use_the_Data_Lab_QueryClient.ipynb

```diff
@@ -1731,17 +1818,33 @@
       "tapdb,splus_edr,nsc_dr1\n",
       "tapdb,splus_edr,sdss_dr14_specobj\n",
       "tapdb,splus_edr,stripe82\n",
+      "tapdb,splus_edr,x1p5__stripe82__allwise__source\n",
+      "tapdb,splus_edr,x1p5__stripe82__gaia_dr2__gaia_source\n",
+      "tapdb,splus_edr,x1p5__stripe82__nsc_dr1__object\n",
+      "tapdb,splus_edr,x1p5__stripe82__sdss_dr16__specobj\n",
+      "tapdb,splus_edr,x1p5__stripe82__unwise_dr1__object\n",
       "tapdb,ivoa_ls_dr8,exposure\n",
       "tapdb,ivoa_ls_dr8,obscore\n",
       "tapdb,ivoa_ls_dr8,siav1\n",
       "tapdb,ivoa_ls_dr8,siav2\n",
+      "tapdb,mydb,123_2659\n",
```

Comment:
I don't know under which user's account you ran this NB, but it looks like it lists many (private?) mydb tables? If this is demo00 user, can we clean up demo00's mydb of all random tables? Thanks!

Reply:
I dropped all tables in demo00's mydb with qc.mydb_drop, but these tables you've pointed to are still there (123_2659, 1_rxs_2491, 20mil_2round_id_ra_dec_2789...). So I'm not sure whose they are or if I have access to remove them?

Reply:
Wait since I used `qc.set_profile('default')` the mydb user must be datalab, right?